### ISSUE TITLE
Upgrade failed: Replay file is not read

### ISSUE THREAD

**Issue by Marenz** [frequenz-floss/frequenz-repo-config-python#134]
### What happened?

Updating to the latest version fails with this error:

```
Traceback (most recent call last):
  File "/tmp/tmp0bhn5nzd.py", line 83, in <module>
    main()
  File "/tmp/tmp0bhn5nzd.py", line 58, in main
    if not NAME_REGEX.match(cookiecutter.name):
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: expected string or bytes-like object, got 'NoneType'
ERROR: Stopping generation because pre_gen_project hook script didn't exit successfully
```

### What did you expect instead?

no error doh

### Affected version(s)

v0.6.1

### Affected part(s)

I don't know (part:❓)

**Comment by llucax** [comment]
Since I won't have time to debug this in the near future (a few weeks) it might be worth mentioning that the workaround is to skip passing the replay file and filling the fields manually. It is annoying but at least there is a workaround.

**Comment by llucax** [comment]
I verified this is an upstream bug. It looks like this got broken by https://github.com/cookiecutter/cookiecutter/pull/1758, released in [cookiecutter 2.2.3](https://github.com/cookiecutter/cookiecutter/releases/tag/2.2.3) and fixed by https://github.com/cookiecutter/cookiecutter/pull/1904 in ]2.3.0](https://github.com/cookiecutter/cookiecutter/releases/tag/2.3.0).

I tested with 2.2.3 and it fails with the error above, 2.2.2 and below and 2.3.0 and 2.4.0 work fine, so basically we only need to avoid the buggy 2.2.3.

Can you please try again using an unproblematic version?

@TalweSingh FYI too.

**Comment by llucax** [comment]
OK, closing as invalid, if you have any issues please reopen.
